### ISSUE TITLE
Fix Joplin department page placeholder text

### ISSUE THREAD

**Issue by desigonz** [cityofaustin/techstack#2368]
### As a content editor, I don't want to accidentally publish out the placeholder text "Mission" and "What we do" on a department page.

**Acceptance criteria**
On department pages:
- [x] "What we do" and "Mission" fields aren't required to publish
- [x] No pre-populated text within the "What we do" and the "mission" fields

------

### background

When you create a new department page, the "What we do" field is pre-populated with the text "What we do", and the "Mission" field is populated with "Mission." This could potentially lead to publishing a page with that text. Also, do we really need this text?

![image.png](https://images.zenhubusercontent.com/5a31a50b8a75884b908b0ea6/4dfdb3e2-8594-4008-8d0f-382d6c5a496a)
![image.png](https://images.zenhubusercontent.com/5a31a50b8a75884b908b0ea6/80ae1ce7-5325-4c2e-b51d-0ff00300a4c1)

Compare with a service page—the "Write here..." fields are grayed out placeholder text that wouldn't publish out if the fields are empty (unlike the department page ones, which *do* publish out)
![image.png](https://images.zenhubusercontent.com/5a31a50b8a75884b908b0ea6/e78886aa-19bf-4a01-94b2-4e03a5ce0cd4)

**Comment by desigonz** [comment]
Per Slack conversation with @toribr: 
- there should be no pre-populated text in the fields, of any color.
- this is a visual bug - see how the form is visible on description but not on the others? it should be visible for all, without any "write here" text.
- @desigonz: Seems like the fields that don't have an outline are the WYSIWYG fields, wonder if that's an out-of-the-box wagtail thing

Might need to write a new issue/rewrite this one.

**Comment by easherma** [comment]
Those fields appear to be pre-populated/filled in as part of the modal dialog, all of which is custom code we wrote and not caused by wagtail.

**Comment by desigonz** [comment]
@easherma yep, totally understand that we wrote the messages—what I'm wondering is if the lack of outline/delineation around the WYSIWYG field is a Wagtail thing.

**Comment by Niicck** [comment]
Elaborating on option 2:
- We could create a custom publish hook that runs our own validation. We'd have a flag on our field that checks if the field is required for publishing vs. draft. Saves a draft with a placeholder key, but doesn't propagate it to the Joplin UI, and doesn't allow publishing. 

This is a bit of a delicate task, but it seems like easiest way forward without losing our required fields. If this is the implementation that we'd be interested in, I'd estimate it at 13.

**Comment by desigonz** [comment]
Thanks @briaguya and @Niicck! A 13 on this doesn't seem worth it for now. We can wait on a more perfect solution when we get to Joplin 3.0.

For now, I think we need to decide what's better between Brian's two options:
1. make them not required, allow users to accidentally publish broken pages with blank "what we do" and "mission" fields
2. keep them required and autopopulating for now, wait to solve this until we have a better solution for different requirements for draft/publish.
@toribr I'm leaning towards option 1, which is how we're dealing with other page types (no pre-populated texts, and we'll have required fields beyond the title once we're at Joplin 3.0).

**Comment by toribr** [comment]
@desigonz Yes, agree that it's the lesser of two evils. :/

**Comment by desigonz** [comment]
@chiaberry to add links to where we can QA and assign to @desigonz

**Comment by chiaberry** [comment]
https://joplin-pr-2368-dept-page-place.herokuapp.com/

**Comment by desigonz** [comment]
@chiaberry Looks great! QA complete.
